SINK-SALES.md

```diff
@@ -0,0 +1,21 @@
+# Entur Sales
```

Comment:
Maybe we could create a folder `src/Docs` in which these files can live, so that we don't clutter the root of the repo?

Reply:
Agreed, fixed